app/views/items/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>All Items</h1>
+
+<% @items.each do |item| %>
+  <section id="test-items-index-<%= item.id %>">
+    <p>Name: <%= link_to item.name, item_path(item) %></p>
+    <p>Price: $<%= item.price %></p>
+    <p>Description: <%= item.description %></p>
+    <p><%= item.image %></p>
```

Comment:
Let's pull this down on your branch and figure out the default image issue now.

Reply:
Also, this shouldn't be a `<p>`, we need to make it an actual image using the `image_tag` action view helper.